From: Simon McVittie <smcv@debian.org>
Date: Sun, 26 Jun 2016 16:56:42 +0100
Subject: test-basic: do not fail in non-English locales

One of the variations tested on Debian's reproducible build
infrastructure is that the second build is done in a French locale.
This test fails there, because it doesn't see "Usage:" in the help.

Signed-off-by: Simon McVittie <smcv@debian.org>
Applied-upstream: 0.6.7, commit:https://github.com/flatpak/flatpak/commit/a41ecc52540cd78fc457456a830babb9de0f8e2f
---
 tests/test-basic.sh | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/tests/test-basic.sh b/tests/test-basic.sh
index f83143e..bc08595 100755
--- a/tests/test-basic.sh
+++ b/tests/test-basic.sh
@@ -21,6 +21,9 @@ set -euo pipefail
 
 . $(dirname $0)/libtest.sh
 
+# This test looks for specific localized strings.
+export LC_ALL=C
+
 echo "1..3"
 
 ${FLATPAK} --version > version_out
